### ISSUE TITLE
Material Design Theme

### ISSUE THREAD

**Issue by manishbisht** [phpmyadmin/phpmyadmin#11811]
Hello Everyone.

I want to contribute on phpmyadmin. Can we implement Material Design Theme in phpmyadmin. It will help to improve the User Interface and making the design more responsive for mobile devices.

Hope you all will like the idea.

Thanks :)

**Comment by ibennetch** [comment]
Hello and welcome.

While we welcome new contributors and especially new themes, the implementation of a mobile-friendly design is probably going to be rather complicated and require some planning and discussion first. I suggest you post to the mailing list with some of your ideas (perhaps even some mockup screenshots) or post some early design work to a pull request for us to review. I'd dislike for you to spend a lot of time on something that won't get implemented, so I think the best way to avoid that is to first discuss your ideas on the mailing list. 

Thanks for your interest!
Isaac

**Comment by zaverichintan** [comment]
I have faced this issue. Would like to help you out, we can togather make some sample screen shots.

**Comment by manishbisht** [comment]
@zaverichintan Can we start working together :)

**Comment by zaverichintan** [comment]
Sure, let's wait for the Approval.

**Comment by isuruAb** [comment]
I would like to contribute on this enhancement

**Comment by NitroBAY** [comment]
I could contribute in some ways also maybe ? I can use materialize css (I try some and it's my favourite...)

**Comment by shuklank2709** [comment]
I would also like to contribute on this idea.But I would like to know where to find mailing list?

**Comment by ibennetch** [comment]
@shuklank2709: See https://www.phpmyadmin.net/support/

**Comment by ibennetch** [comment]
For everyone who still wishes to work on this idea, the best way would be to start a mailing list thread where you discuss how you plan to overcome some of the problems (such as fitting all the menu choices in to a phone/table screen size or how to paginate 10,000 responses on a small screen) as well as an overview of the code chances that would be required -- if there are any base code changes on top of designing a modified theme. These are the fundamental difficulties and decisions that this project will face, so starting discussion about them is the first step.

**Comment by zaverichintan** [comment]
RTL support on mobile phone.  Is that possible with material design?

**Comment by philippeboyd** [comment]
I'll just leave this here
http://www.materialup.com/posts/phpmyadmin-material-design

**Comment by ibennetch** [comment]
The link posted by @philippeboyd does look good but doesn't look like anything more than a custom designed theme to me. We're always welcome to receive new theme submissions.

**Comment by NitroBAY** [comment]
Yes I watch this since months, indeed it's just a design, not a project for now but could be... I think you should contact Augustin, it's rly good

**Comment by manishbisht** [comment]
@ibennetch Can i work on this theme ?

**Comment by yadavankit** [comment]
@ibennetch I would also love to work on this Theme. Can't ignore how awesome it would be to implement for PMA. Well, I have worked on materialize CSS before.

**Comment by ibennetch** [comment]
I wish to stress again that, as great as new theme development is and as much as we appreciate our theme contributors, theme development itself has not been an acceptable GSoC project in years past, according to Google. Working our code base in to something that degrades well to a mobile/tablet screen *may* qualify (due to the code changes required), but I'm not sure at the moment and have doubts.

As far as the pre-application code submission we request to show a potential student has some understanding of the tools used by the phpMyAdmin project, theme development will likely put a potential applicant ahead of a student who has not submitted any code but probably will not look as good on the application as bug fixes, features added, or other changes to the codebase itself.

Hope that makes sense.

**Comment by ibennetch** [comment]
With that being said, @manishbisht has been interested in this item for some time, so yes you may work on it, if you wish. @yadavankit, any theme you develop will probably look different from @manishbisht's so your contribution is welcome as well.

As a reminder, you can add a pull request against the theme tracker at https://github.com/phpmyadmin/themes/pulls for new themes.

**Comment by manishbisht** [comment]
Thanks @ibennetch Now, I will start working on this theme :)

**Comment by mauriciofauth** [comment]
Material Design uses the Apache License 2.0, I don't know if this license is compatible with the GPLv2. Does anyone know more about this?

**Comment by akshil4444** [comment]
Hi, I want to contribute to this project as a part of GSoC 2018 
why don't we use bootstrap for now and later on shift to the material design?
